llvm/unittests/Analysis/TargetLibraryInfoTest.cpp

```diff
@@ -318,6 +318,7 @@ TEST_F(TargetLibraryInfoTest, ValidProto) {
       "declare void @qsort(i8*, i64, i64, i32 (i8*, i8*)*)\n"
       "declare i64 @readlink(i8*, i8*, i64)\n"
       "declare i8* @realloc(i8*, i64)\n"
+      "declare i8* @reallocarray(i8*, i64, i64)\n"
```

Comment:
This test should be converted to opaque pointers

Reply:
ack. but that probably belongs to another commit, right?

Reply:
yes